### PR TITLE
Send error to rollbar if packages need updating

### PR DESCRIPTION
**Pull request by patiee**

### CODE REVIEW

**Review by lukanus — APPROVED** [review]
Without bigger context for me looks ok, I hope it'll tell more to @pacmessica

File: utils/upgrade_alert.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+    const upgraded = await ncu.run({
+      packageFile: 'package.json',
+      jsonUpgraded: true,
+      silent: true,
+      upgrade: true,
+      filter: filter
+    })
+
+    if (JSON.stringify(upgraded) !== "{}") {
```
Comment (pacmessica) [inline utils/upgrade_alert.js:29]: Does this not work `Object.keys(upgraded).length === 0`?

**Review by pacmessica — APPROVED** [review]
lgtm

File: utils/upgrade_alert.js (hunk 1 above)
Comment (patiee) [reply utils/upgrade_alert.js:28]: unfortunately no :/

**Pull request merged**